cockatrice/src/window_main.cpp

```diff
@@ -107,12 +107,15 @@ void MainWindow::processConnectionClosedEvent(const Event_ConnectionClosed &even
         case Event_ConnectionClosed::LOGGEDINELSEWERE: reasonStr = tr("You have been logged out due to logging in at another location."); break;
         default: reasonStr = QString::fromStdString(event.reason_str());
     }
-    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.\nReason: %1").arg(reasonStr));
+    QMessageBox::critical(this, tr("Connection closed"), tr("The server has terminated your connection.") + QString("<br>")
```

Comment:
Won't the `<br>` in here allow a server operator to inject html+javascript on you as ctrl mentioned? We shouldn't use `<br>` with any dynamic strings